src/firefly/js/fieldGroup/FieldGroupUtils.js

```diff
@@ -133,6 +133,10 @@ export const getFieldGroupState= function(groupKey) {
     return fieldGroupMap[groupKey] ? fieldGroupMap[groupKey] : null;
 };
 
+export function getFieldVal(groupKey, fldName, defval=undefined) {
+    return get(getGroupFields(groupKey), [fldName, 'value'], defval);
+};
+
```

Comment:
getFldValue expects fields and fieldName.  In order to get fields, you need to call getGroupFields with groupKey.  
The above function is a convenience way to get the value using just one import, groupKey, and fieldName.  getFldValue is used everywhere and is more optimized when used multiple times.